commons/src/build_output.rs

```diff
@@ -533,14 +533,15 @@ pub mod fmt {
         let noun = noun.as_ref();
         let header = header.as_ref();
         let body = help_url(body, url);
```

Comment:
These output changes seem unrelated? If so, I think they should be in their own PR :-)